content/applications/productivity/discuss/canned_responses.rst

```diff
@@ -0,0 +1,123 @@
+================
+Canned responses
+================
+
+*Canned responses* are customizable inputs where a typed shortcut populates a longer response. A
+user enters a keyword shortcut, which is then automatically replaced by the expanded substitution
+response. Canned responses save time by allowing users to use shorthand phrases to populate longer
+messages. This also limits the possibility of errors when typing out longer messages because these
+are pre-set messages. This maintains consistency throughout customer interactions.
+
+Canned responses consist of two main components: the *shortcut* and the *substitution*. The shortcut
+is the keyword or key phrase that is to be replaced. The substitution is the longer message that
+replaces the shortcut.
+
+.. image:: canned_responses/canned-response-sample.png
+   :align: center
+   :alt: A live chat conversation using a canned response.
+
+Canned responses are available :ref:`to use <discuss/use-cases>` in *Live Chat* conversations, the
+*Discuss* app, and the *Chatter* composer. This includes direct message conversations, channel
+conversations, and *WhatsApp* messages.
+
+.. _discuss/created-canned-response:
+
+Creating canned responses
+=========================
+
+Canned responses are managed through the *Discuss* application. To create a new canned response, or
+to manage the list of existing responses, navigate to :menuselection:`Discuss app --> Configuration
+--> Canned Responses`.
+
+To create a new canned response, click :guilabel:`New` at the top-left of the list.
+
+Canned responses consist of two main components, a *shortcut* the user enters, and the
+*substitution* that replaces the shortcut.
+
+.. image:: canned_responses/shortcut-substitution.png
+   :align: center
+   :alt: A list of canned responses emphasizing the shortcut and substitution fields.
+
+Type a shortcut command in the :guilabel:`Shortcut` field. Next, click on the
+:guilabel:`substitution` field and type the message that will replace the shortcut.
+
+.. tip::
+   Try to connect the shortcut to the topic of the substitution. Not only does this make it easier
+   to use the responses, it prevents the list of responses from becoming disorganized and
+   overwhelming.
+
+In the :guilabel:`Description` field, add any information that provides context for this response,
+such as guidelines for when it should or should not be used.
+
```

Comment:
is it worth mentioning the 'Created by' column? just to kinda round things out? Even though you can't edit this field, it might be worth a quick mention or admonition, so the reader feels like they're getting a "complete overview" of the Canned Responses page columns